components/partial/code/CodeExamples.tsx

```diff
@@ -44,15 +30,31 @@ const getLanguageOptions = (language: string, ctxLanguageOptions) =>
 const codeStringFrom = (codeExample: any): any =>
   Children.only(codeExample.props.children).props.children;
 
+type CodeExampleProps = { language: Language; children: string };
+
+// eslint-disable-next-line no-unused-vars
+export const CodeExample = (props: CodeExampleProps) => null;
```

Comment:
This component basically just exists so we can grab the props off it. In `toCodeExampleProps` below, it takes in a `MDXCreateElement` or `CodeExample` component and returns the props.